README.md

```diff
@@ -34,45 +40,44 @@ $ cmake .. \
 # -DBUILD_TESTING=ON
 $ make install
 ```
-where lines need to be commented in and the paths modified if cmake cannot
-(1) find MATLAB, (2) find FFTW or (3) install to the default install prefix.
-
-- <details>
-    <summary>Mac specific instructions</summary>
-
-    To compile on a Mac an x86 compiler with libraries for OpenMP are required,
-    which can be installed using [brew](https://brew.sh/) with `brew install llvm`
-    then (optionally) set the following cmake arguments
-
-    ```
-    -DCMAKE_CXX_COMPILER=/Users/username/.local/homebrew/opt/llvm/bin/clang++
-    -DOMP_ROOT=/Users/username/.local/homebrew/opt/llvm/
-    -DCXX_ROOT=/Users/username/.local/homebrew/opt/llvm
-    ```
-
-    On an ARM Mac install the x86 version of brew with
-    ```bash
-    arch -x86_64 zsh
-    arch -x86_64 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-    arch -x86_64 /usr/local/bin/brew install llvm
-    ```
+where lines need to be commented in and the paths modified if cmake cannot (1)
+find MATLAB, (2) find FFTW or (3) install to the default install prefix.
+
+<details>
+<summary>Mac specific instructions</summary>
+
+To compile on a Mac an x86 compiler with libraries for OpenMP are required,
+which can be installed using [brew](https://brew.sh/) with `brew install llvm`
+then (optionally) set the following cmake arguments
+
+```{sh}
```

Comment:
```suggestion
```bash
```

Minor point, but why `{sh}` here and then `bash` on line 60? Are they subtlety different in how/what they render/highlight?

Reply:
Ah, that's the [doxygen-flavoured markdown way to get syntax highlighting](https://doxygen.nl/manual/markdown.html#md_fenced). Probably this means that on the web page, the line 60 code block is not highlighted.

Although as it's anyway bash, it's probably a bit of a moot point.

Reply:
Looks like GitHub is smart enough to understand both. I'll check when I go over these to check for info duplication.